### PR TITLE
Use poetry

### PR DESCRIPTION
**Pull request by myakove**

### CODE REVIEW

**Comment by myakove** [comment]
The following are automatically added:
 * Add reviewers from OWNER file (in the root of the repository) under reviewers section.
 * Set PR size label.
 * New issue is created for the PR.

Available user actions:
 * To mark PR as verified add `/verified` to a PR comment, to un-verify add `/verified cancel` to a PR comment.
        Verified label removed on each new commit push.
 * To cherry pick a merged PR add `/cherry-pick <target branch to cherry-pick to>` to a PR comment.
 * To add a label by comment use `/<label name>`, to remove, use `/<label name> cancel`
  Supported labels:
  * hold
  * verified
  * wip
  * lgtm
  * approve
  * target-version-

File: webhook_server_container/app.py [hunk 1]
```diff
@@ -40,3 +42,16 @@ def process_webhook():
...
+async def main():
+    await create_webhook(app=app)
+    app.logger.info("Starting webhook-server app")
+    app.run(port=5000, host="0.0.0.0", use_reloader=False)
+
+
+loop = asyncio.get_event_loop()
+loop.run_until_complete(main())
+
+# if __name__ == "__main__":
```
Comment (rnetser) [inline webhook_server_container/app.py:56]: should be removed?

**Review by rnetser — APPROVED** [review]

**Pull request merged**